modules/csv/csv_test.go

```diff
@@ -21,14 +22,23 @@ func TestCreateReader(t *testing.T) {
 	assert.Equal(t, ',', rd.Comma)
 }
 
-//nolint
+func decodeSlashes(s string) string {
+	s = strings.ReplaceAll(s, "\n", "\\n")
+	s = strings.ReplaceAll(s, "\"", "\\\"")
+	decoded, err := strconv.Unquote(`"` + s + `"`)
+	if err != nil {
+		panic("unable to decode string:" + err.Error() + "\n" + s)
```

Comment:
panic in test sounds like a bad idea!

Reply:
It's in unit test (for helper), so I think it's fine. Otherwise it would make code too complex.

Reply:
just pass `t *testing.T` as argument & t.FailNow()

Reply:
I prefer not, the `decodeString` is not part of the unit test ........ the panic should never happen

Reply:
Hmm, finally, I updated to `func decodeSlashes(t *testing.T, s string) string`

Reply:
I just have seen to many `panic`s in legacy code that wen evil at some point as code changes

Reply:
thanks :+1: